pgdrive/utils/asset_loader.py

```diff
@@ -9,11 +11,14 @@ class AssetLoader:
     asset_path = None
```

Comment:
So do we still need the static method here? Let's remove it.

We should collect a `root_path` when importing the asset_loader (just like what we do before in the top of each file using `__file__` to get the root. ), rather than requesting the user to all the static `init_loader` all the time.

Reply:
I treat AssetLoader as a global variable, which initializes only once and only in the ```pgworld.__init__()```. It is ok to collect a root_path when importing the asset_loader. But, AssetLoader also keeps a reference to ShowBase.ShowBase.loader, after calling ```init_loader```. To do so, elements can load assets without accessing ```PgWorld.loader```. Otherwise, all elements must add a new parameter in  their ```__init__()``` function. So the ```init_loader``` can not be removed.

Reply:
I have push a new PR. Would you please try to implement several helper function that can do what AssetLoader do without accessing the class AssetLoader?

I think this would be more natural. Just like `ray.init()` is a function, but it register a worker in the background.

Reply:
I have replaced all ```AssetLoader.init_loader()``` with ```initialize_asset_loader()```.  As for other helper function, do you mean implementing all staticmethods out of the AssetLoader class? And the AssetLoader class only maintains the ```root_path``` and ```ShowBase.loader```? I don't know the meaning of doing this....